pkg/odo/cli/debug/portforward.go

```diff
@@ -45,10 +45,7 @@ type PortForwardOptions struct {
 }
 
 var (
-	portforwardLong = templates.LongDesc(`
-			Forward a local port to a remote port on the pod where the application is listening for a debugger.
-
-			By default the local port and the remote port will be same. To change the local port you can use --local-port argument and to change the remote port use "odo config set DebugPort <port>"   		  
+	portforwardLong = templates.LongDesc(`Forward a local port to a remote port on the pod where the application is listening for a debugger. By default the local port and the remote port will be same. To change the local port you can use --local-port argument and to change the remote port use "odo config set DebugPort <port>"   		  
```

Comment:
For devfile components, `config set` doesn't work.
```
[mrinaldas@localhost project]$ odo config set DebugPort 5858
 ✗  unknown parameter :'debugport' is not a configurable parameter in the devfile
```
It has been changed to `env set` for devfile components.

Reply:
Thanks! Updated.